src/pages/wallet-details/wallet-details.html

```diff
@@ -44,7 +44,7 @@
           </div>
         </div>
 
-        <div class="balance-spendable" (click)="openBalanceDetails()" *ngIf="!wallet.balanceHidden && !wallet.scanning && showBalanceButton && wallet.status">
```

Comment:
If you delete showBalanceButton, probably you have to remove that variable from the controller file also